test/remote/account_create_tests.js

```diff
@@ -35,19 +34,24 @@ TestServer.start(config)
             t.fail('got keys before verifying email')
           },
           function (err) {
-            keyFetchToken = client.keyFetchToken
-            t.ok(client.keyFetchToken, 'retained keyFetchToken')
-            t.equal(err.message, 'Unverified account', 'account is unverified')
-          }
-        )
-        .then(
-          function () {
-            return client.emailStatus()
+            t.equal(err.errno, 104, 'Unverified account error code')
+            t.equal(err.message, 'Unverified account', 'Unverified account error message')
           }
         )
+    }
+  )
+
+  test(
+    'create and verify account',
+    function (t) {
+      var email = server.uniqueEmail()
+      var password = 'allyourbasearebelongtous'
+      var client = null
+      return Client.create(config.publicUrl, email, password)
         .then(
-          function (status) {
-            t.equal(status.verified, false)
```

Comment:
I don't mind leaving it out, just as long as we're cross-checking that we don't lose coverage on anything